packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -411,6 +440,22 @@ export class AppMainContainer extends Component<
     this.emitLayoutEvent(InputFilterEvent.CLEAR_ALL_FILTERS);
   }
 
+  sendCycleStackForward(): void {
```

Comment:
See my other change about refactoring to NavigationEvent, then instead of calling `this.emitLayoutEvent(...)` it'll just be `emitCycleToNextStack(this.layout)`